2025-03-17 03:55:59,259 - EmulValidation.py - 33 - DEBUG - scripts.EmulValidation - =============================test starts here========================
2025-03-17 03:56:23,153 - vex_boot_cs_test.py - 121 - INFO - __main__ - 0
2025-03-17 03:56:23,154 - vex_boot_cs_test.py - 122 - INFO - __main__ - 1000242040
2025-03-17 03:56:23,159 - EmulValidation.py - 78 - ERROR - scripts.EmulValidation - there are  errors in the test, please resolve
2025-03-17 03:56:23,159 - EmulValidation.py - 79 - ERROR - scripts.EmulValidation - no of errors are 1
2025-03-17 03:56:23,159 - EmulValidation.py - 80 - ERROR - scripts.EmulValidation - ========================= test Fail =============================
2025-03-17 03:56:23,159 - vex_boot_cs_test.py - 161 - INFO - __main__ - ================================test run completed==============================
